ecl2df/eclfiles.py

```diff
@@ -146,7 +146,15 @@ def get_eclsum(self, include_restart=True):
                     errno.ENOENT, os.strerror(errno.ENOENT), smryfilename
                 )
             logger.info("Opening UNSMRY file: %s", smryfilename)
-            self._eclsum = EclSum(smryfilename, include_restart=include_restart)
+            try:
+                self._eclsum = EclSum(smryfilename, include_restart=include_restart)
+            except IOError:
+                # This can happen if there is something wrong with the file
+                # or if SMSPEC is missing.
+                logger.warning(
+                    "Failed to create summary instance from %s", smryfilename
```

Comment:
Why is a failure a warning and not an actual failure?

Reply:
Right, degrading the failure should not happen here, but in fmu.ensemble if it wants to. Changing to raise.. from